src/libstore/build/derivation-goal.cc

```diff
@@ -604,7 +604,7 @@ void DerivationGoal::inputsRealised()
                     return *outPath;
                 }
                 else {
-                    auto outMap = worker.evalStore.queryDerivationOutputMap(depDrvPath);
```

Comment:
Small change to CA behavior. Ignores `store` from now on.
This is in an experimental feature, relating to an already known-broken non-experimental feature.
@Ericson2314 continues to work on this.